templates/provider/cluster-api-provider-openstack/templates/orc-1-0-0.yaml

```diff
@@ -1,856 +0,0 @@
-{{ if (eq .Values.orcVersion "1.0.0") }}
```

Comment:
If `orcVersion` was explicitly set to `1.0.0` in the management spec (I know this was the default value and overriding it might not make sense, but this scenario can still occur), and CAPO was upgraded from 0.3.1 to 0.3.2, all orc components will be removed because the condition no longer matches.
Wouldn’t it make sense to keep the support for `orc-1-0-0` as well?

Reply:
It makes sense in that way. I would rather rework the whole approach, completely removing this knob, otherwise we'll end with dozens of manifests for the BW compatibility.

My idea is:
1 remove the knob (`orcVersion`)
2 drop the `if` statement (since there won't be such a value)
3 pin the orc version in the `makefile`

File names might still be versioned to be explicit about the current versioning 

Let me know what you think

Reply:
Of course, the proposed solution implies that a potential user of the `orcVersion` value must remove it. But I would embrace it that was,y having (almost) no users of the value

Reply:
I think it makes sense. Let's proceed with the new approach.

Reply:
So I've preserved the 1.0.0 manifest, added a new one without any changes to it, namely no `if` statements and placeholders